### PR TITLE
Add fetch method

### PR DESCRIPTION
**Pull request by hananeikhelef**
- Add a button to the About page that fetches comments from the serve

### CODE REVIEW

File: portfolio/src/main/webapp/contact.html [hunk 1]
```diff
@@ -14,10 +14,9 @@
             <div class="logo_container">
                <h1> <span> Hanane Ikhelef  </span></h1>
                <ul class="navigation">
-                    <li>  About
-                        <a href= "index.html"> </a>
-                     </li>
-            
+                  <li>  About
+                     <a href= "index.html"> </a>
+                  </li>
```
Comment (mctavish) [inline portfolio/src/main/webapp/contact.html:19]: This is fine, but generally you'll want to make sure that the only changes in a PR are the ones related to the feature that you're working on.  If we were to be hugely proper, the reformatting would be in its own independent PR.  This way the reformatting can be reviewed and approved quickly and easily, and the feature PR is smaller and more focused.

File: portfolio/src/main/webapp/contact.html [hunk 2]
```diff
@@ -44,6 +43,10 @@ <h2 onClick="toggleClass()">Get in touch with me or leave a comment   <span>+</s
                </div>
             </div>
          </form>
+         <div id="content">
+         </div>
```
Comment (mctavish) [inline portfolio/src/main/webapp/contact.html:47]: Nit: Looks better if it's on the same line.

File: portfolio/src/main/webapp/contact.html [hunk 3]
```diff
@@ -44,6 +43,10 @@ <h2 onClick="toggleClass()">Get in touch with me or leave a comment   <span>+</s
...
+         <button onclick="getMessage()"> See other comments !</button>
```
Comment (mctavish) [inline portfolio/src/main/webapp/contact.html:48]: Nit: No space before punctuation.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -20,3 +20,9 @@ function toggleClass(){
 	var element = document.getElementById('contact');
 	element.classList.toggle("active")
 }
+
+function getMessage() {
+  fetch('/data').then(response => response.text()).then((message) => {
+    document.getElementById('message-container').innerText = message;
+  });
```
Comment (mctavish) [inline portfolio/src/main/webapp/script.js:27]: This can be collapsed into a single .then() (containing multiple statements in the code block) which will make it easier to read.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -20,3 +20,9 @@ function toggleClass(){
 	var element = document.getElementById('contact');
 	element.classList.toggle("active")
 }
+
+function getMessage() {
```
Comment (mctavish) [inline portfolio/src/main/webapp/script.js:24]: Please add documentation for the intended functionality of this code.

File: portfolio/src/main/webapp/contact.html (hunk 1 above)
Comment (hananeikhelef) [reply portfolio/src/main/webapp/contact.html:19]: yeah, my bad! 
When i save the file the HTML  auto-formatter makes changes in the file.

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (hananeikhelef) [reply portfolio/src/main/webapp/script.js:26]: Done !
